### DISCUSSION TITLE
What do I need to do inorder to get the samples to build

### DISCUSSION THREAD

**Discussion by SteveSitekitcare** [shinyorg/shiny#1348] [Q&A]
Hi

I'm probably being stupid, but Ive pulled the source in order to take a look at the maui push sample. But I cant seem to get the src to build. Is there a set of instructions/requirements in order to make it work?

Ive attached the errors, just incase they are of interest
[ShinyErrors.txt](https://github.com/shinyorg/shiny/files/13302245/ShinyErrors.txt)

Cheers

**Comment by aritchie** [comment]
The samples are meant more of a guide vs runnable the code.  You have to be able to build all of the libraries and I can see you're doing this from a windows machine, Shiny on windows, will try to compile with old Xamarin as well which appears is not installed on your machine:

> c:\nuget\msbuild.sdk.extras\3.0.44\Build\LanguageTargets\CheckMissing.targets(44,5): error : The specified language targets for xamarin.ios10 is missing. Ensure correct tooling is installed for 'xamarin.ios'. Missing: 'C:\Program Files\Microsoft 

Secondly, push requires you setup things like your provisioning profile setup for push, bundle ids, push certs, all that junk just to test - which is again - another reason the samples are more guides than anything
